cmd/bootkube/main.go

```diff
@@ -13,9 +13,10 @@ import (
 
 var (
 	cmdRoot = &cobra.Command{
-		Use:   "bootkube",
-		Short: "Bootkube!",
-		Long:  "",
+		Use:           "bootkube",
+		Short:         "Bootkube!",
+		SilenceErrors: true,
```

Comment:
Add a comment? Does this work for subcommands too?

Reply:
sure does... it chains.

Reply:
added a comment as well.